public/v4/apps/appwrite.yml

```diff
@@ -0,0 +1,1338 @@
+captainVersion: 4
+
+services:
+    $$cap_appname-redis:
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+        image: 'bitnami/redis:6.0.8-debian-10-r6'
+        ports:
+            - '6379:6379'
+        environment:
+            REDIS_DISABLE_COMMANDS: FLUSHDB,FLUSHALL
+            ALLOW_EMPTY_PASSWORD: 'yes'
+        volumes:
+            - $$cap_appname-redis-data:/bitnami/redis/data
+
+    $$cap_appname-mariadb:
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+        image: 'docker.io/bitnami/mariadb:10.7'
+        ports:
+            - '3306:3306'
+        environment:
+            MARIADB_ROOT_PASSWORD: $$cap_APP_DB_ROOT_PASS
+            MARIADB_DATABASE: $$cap_APP_DB_SCHEMA
+            MARIADB_USER: $$cap_APP_DB_USER
+            MARIADB_PASSWORD: $$cap_APP_DB_PASS
+        command: 'mysqld --innodb-flush-method=fsync'
```

Comment:
CapRover doesn't consume `command` as per docs here: https://github.com/caprover/one-click-apps#services

Either remove or convert to `dockerfileLines` if needed (search repo for examples)

Reply:
I used dockerfileLines properly. I forgot about it. 😞